katastertools/sql/popisne-data.sql

```diff
@@ -23,7 +23,7 @@ BEGIN;
 		oce integer,
 		cas character varying(30),
 		crc integer,
-		naz_suboru character varying(20)
```

Comment:
lebo všetky grafické dáta majú atribút 'subor', ktorý je ok, ale všetky popisné dáta majú atribút 'naz_suboru', ktorý je vo všetkých importovaných tabuľkách ako NULL .. ale je to pravda, ani týmto sa to nevyriešilo